docs/en/customization/build.md

```diff
@@ -9,20 +9,18 @@ for their particular project.
 - forking existing ones to make changes,
 - or building a lightweight server with only the necessary plugins.
 
-To facilitate this process, we have developed the **Velox** tool, which enables developers to build a RoadRunner server
-binary based on a configuration file specifying the required plugins and repositories for a specific project.
+We created a tool called **Velox** that lets developers build a RoadRunner server binary. They do this by using a
+configuration file that tells the tool which plugins and repositories are needed for the project.
 
 ## Configuration
 
-The configuration file, written in TOML format, specifies the list of repositories to include in the build, along with
-the respective owners and versions. It also allows the inclusion of private repositories hosted on both Github and
-Gitlab, with authentication via access tokens.
+The configuration file is written in TOML format and contains a list of repositories to add to the build. For each
+repository, you can specify the owner and version. You can also add private repositories from Github or Gitlab, and
+authenticate with access tokens.
```

Comment:
The developer needs GitHub token in any case (to download all plugins). Anonymous access have a limit of the 60 req per hour IIRC.